WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -197,47 +182,46 @@ private extension SupportTableViewController {
         }
 
         // Create and return table
-        let sections = [helpSection, forumsSection, informationSection, logOutSections].compactMap { $0 }
+        let sections = [forumsSection, informationSection, logOutSections].compactMap { $0 }
         return ImmuTable(sections: sections)
     }
 
-    // TODO - remove after FeatureFlag.wordPressSupportForum is removed
-    func oldTableViewModel() -> ImmuTable {
+    func jetpackAppTableViewModel() -> ImmuTable {
 
         // Help Section
         var helpSectionRows = [ImmuTableRow]()
-        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button", featureFlagSupportForum: featureFlagStore.value(for: FeatureFlag.wordPressSupportForum)))
+        helpSectionRows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button", featureFlagSupportForum: isForumShown))
```

Comment:
@guarani parameter `featureFlagSupportForum` for class `HelpRow` no longer makes sense here since this parameter was used to control the text color depending on feature flag state.  Thanks for those screenshots on before and after feature flag; since the colour is no longer changing based on feature flag this is redundant now.  Thanks!

Reply:
Gotcha! I saw those feature flags being passed in but wasn't sure it was relevant here, but your explanation makes sense. I've pushed a change to remove these.